zerver/lib/import_realm.py

```diff
@@ -1238,6 +1235,15 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
         update_model_ids(GroupGroupMembership, data, "groupgroupmembership")
         bulk_import_model(data, GroupGroupMembership)
 
+        re_map_foreign_keys(
+            data, "zerver_usergroup_can_mention_groups", "from_usergroup", related_table="usergroup"
+        )
+        re_map_foreign_keys(
+            data, "zerver_usergroup_can_mention_groups", "to_usergroup", related_table="usergroup"
+        )
+        update_model_ids(UserGroup.can_mention_groups.through, data, "usergroup_can_mention_groups")
+        bulk_import_model(data, UserGroup.can_mention_groups.through)
```

Comment:
This can probably still be written as a loop over `UserGroup.GROUP_PERMISSION_SETTINGS`; and similarly with the export logic; probably a good idea to do that.